### PR TITLE
JCT-147: update SCIM documentation how to deactivate user

### PR DESCRIPTION
**Pull request by Veki301**
## Checklist

 - [ ] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [ ] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: docs/src/understand/single-sign-on/main.rst [hunk 1]
```diff
@@ -546,6 +546,10 @@ For each put request, you need to provide the full json object.  All omitted fie
      -d "$SCIM_USER" \
      $WIRE_BACKEND/scim/v2/Users/$STORED_USER_ID
 
+**Deactivate user**
+
+It is possible to temporarily deactivate an user (and reactivate him later) by setting his ``active`` property to ``true/false`` without affecting his device history.
```
Comment (fisx) [inline docs/src/understand/single-sign-on/main.rst:551]:
```suggestion
It is possible to temporarily deactivate an user (and reactivate him later) by setting his ``active`` property to ``true/false`` without affecting his device history.  (`active=false` changes the wire user status to `suspended`.)
```

**Review by fisx — APPROVED** [review]
Thanks!  Could you also add a changelog file under `changelog.d/4-docs/pr-2720` that contains just the string "Document user deactivation (aka suspension) with SCIM."?

I would do it myself, but it's your fork.  :)

**Review by fisx — CHANGES_REQUESTED** [review]
see above comments.

**Comment by Veki301** [comment]
@fisx added changelog, please merge once checks are done, thank you :)

**Review by fisx — APPROVED** [review]

**Pull request merged**